frontend/lib/providers/filtered_family_artefacts.dart

```diff
@@ -1,28 +1,42 @@
+import 'dart:collection';
+
 import 'package:dartx/dartx.dart';
 import 'package:riverpod_annotation/riverpod_annotation.dart';
 
 import '../models/artefact.dart';
 import '../models/filters.dart';
 import '../routing.dart';
+import '../utils/artefact_sorting.dart';
 import 'family_artefacts.dart';
 
 part 'filtered_family_artefacts.g.dart';
 
 @riverpod
-Map<int, Artefact> filteredFamilyArtefacts(
+LinkedHashMap<int, Artefact> filteredFamilyArtefacts(
```

Comment:
TIL: `LinkedHashMap` is the sorted version of a `Map`, really useful in such scenario!